<ui version="4.0" >
 <class>Frame</class>
 <widget class="QWidget" name="Frame" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>20</height>
   </rect>
  </property>
  <property name="maximumSize" >
   <size>
    <width>16777215</width>
    <height>20</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>AttribInt</string>
  </property>
  <property name="title" stdset="0" >
   <string>AttribInt</string>
  </property>
  <property name="flat" stdset="0" >
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label" >
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text" >
      <string>AtrribInt</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="value" />
   </item>
   <item>
    <widget class="QPushButton" name="prevButton" >
     <property name="text" >
      <string>Prev</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="nextButton" >
     <property name="text" >
      <string>Next</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
